### PR TITLE
Create midyear-performance-review-timeline-2020

### PR DESCRIPTION
**Pull request by lynn915**

### CODE REVIEW

File: operations/workplace/people/performance-reviews-50/midyear-performance-review-timeline-2020 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## R&D Mid-Year Performance Review Timeline 2020
+
+May 25: Performance evals kick-off
+May 25 - 28: Managers to provide peer panels to HR 
+May 29: HR sends Self-Assessments (Bamboo) & Peer Feedback Surveys (Survey Monkey)
```
Comment (justinegeffen) [inline operations/workplace/people/performance-reviews-50/midyear-performance-review-timeline-2020]:
```suggestion
May 29: HR sends Self-Assessments (Bamboo) and Peer Feedback Surveys (Survey Monkey)
```

**Review by natalie-hub — DISMISSED** [review]

File: operations/workplace/people/performance-reviews-50/midyear-performance-review-timeline-2020 [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## R&D Mid-Year Performance Review Timeline 2020
+
+May 25: Performance evals kick-off
```
Comment (justinegeffen) [inline operations/workplace/people/performance-reviews-50/midyear-performance-review-timeline-2020]:
```suggestion
May 25: Performance Evaluations kick-off
```

File: operations/workplace/people/performance-reviews-50/midyear-performance-review-timeline-2020 [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+May 25 - 28: Managers to provide peer panels to HR 
+June 1: HR sends Self-Assessments (Bamboo) and Peer Feedback Surveys (Survey Monkey)
+June 12: All Evaluations due
+June 17: HR shares Peer Feedback Surveys with Managers via Bamboo
+June 17 - 26: Managers complete Manager Feedback in Bamboo. Staff members receive evals in Bamboo automatically once Manager Feedback is submitted.
```
Comment (justinegeffen) [inline operations/workplace/people/performance-reviews-50/midyear-performance-review-timeline-2020]:
```suggestion
June 17 - 26: Managers complete Manager Feedback in Bamboo. Staff members receive evaluations in Bamboo automatically once Manager Feedback is submitted.
```

File: operations/workplace/people/performance-reviews-50/midyear-performance-review-timeline-2020 [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+Managers to schedule time with each staff members to discuss their eval.
```
Comment (justinegeffen) [inline operations/workplace/people/performance-reviews-50/midyear-performance-review-timeline-2020]:
```suggestion
Managers to schedule time with each staff member to discuss their evaluation. 
```

**Review by justinegeffen — APPROVED** [review]
I made some minor stylistic changes which I have committed. LGTM. :)

**Pull request merged**